go.mod

```diff
@@ -12,32 +11,31 @@ require (
 	github.com/go-logr/zapr v0.2.0 // indirect
 	github.com/google/go-cmp v0.5.5
 	github.com/googleapis/gnostic v0.5.1 // indirect
-	github.com/imdario/mergo v0.3.10 // indirect
-	github.com/mholt/archiver v2.1.0+incompatible
+	github.com/mholt/archiver/v3 v3.5.0
 	github.com/moby/term v0.0.0-20201110203204-bea5bbe245bf // indirect
-	github.com/nwaples/rardecode v1.1.0 // indirect
 	github.com/onsi/ginkgo v1.14.1 // indirect
 	github.com/onsi/gomega v1.10.2 // indirect
 	github.com/prometheus/client_golang v1.9.0
 	github.com/spf13/afero v1.5.1
 	go.uber.org/zap v1.15.0 // indirect
-	golang.org/x/time v0.0.0-20200630173020-3af7569d3a1e // indirect
 	google.golang.org/appengine v1.6.6 // indirect
-	gotest.tools/v3 v3.0.3 // indirect
-	helm.sh/helm/v3 v3.4.2
-	k8s.io/api v0.19.4
-	k8s.io/apimachinery v0.19.4
-	k8s.io/client-go v0.19.4
-	k8s.io/utils v0.0.0-20200912215256-4140de9c8800 // indirect
-	rsc.io/letsencrypt v0.0.3 // indirect
+	helm.sh/helm/v3 v3.5.3
+	k8s.io/api v0.20.2
+	k8s.io/apimachinery v0.20.2
+	k8s.io/client-go v0.20.2
```

Comment:
The problem is the CAPI forks are still using v0.18.6
https://github.com/giantswarm/cluster-api/blob/v0.3.13-gs/go.mod#L40

Can you try vendoring this in chart-operator? If it compiles and the tests pass this should be fine to merge.

Reply:
Seems to work https://github.com/giantswarm/chart-operator/pull/708

Reply:
Nice, thanks for checking.